ui_testing/testcases/basic_tests/test03_orders.py

```diff
@@ -750,3 +750,65 @@ def test017_update_shipment_date(self, save):
                 ' + Assert {} (current_shipment_date) == {} (order_shipment-date)'.format(current_shipment_date,
                                                                                           order_shipment_date))
             self.assertEqual(current_shipment_date, order_shipment_date)
+
+    @parameterized.expand(['save_btn', 'cancel'])
+    def test020_update_material_type_with_test_unit(self, save):
+        """
+
+        New: Orders: Material type Approach: I can update the material type
+        filed with test units records successfully
+
+        LIMS-3844
+        LIMS-3844
+        :return:
+        """
+
+        self.order_page.get_orders_page()
+        self.order_page.click_create_order_button()
+        order_url = self.base_selenium.get_url()
+        self.base_selenium.LOGGER.info(' + order_url : {}'.format(order_url))
+        self.order_page.set_new_order()
+        self.order_page.set_contact()
+        self.order_page.set_material_type(
+            material_type='r')
+        self.order_page.set_article(
+            article='a')
+        self.order_page.set_test_unit(
+            test_unit='f')
+        self.order_page.save(save_btn='order:save_btn')
```

Comment:
replace from 767 to 787 with `self.order_page.create_new_order(material_type='r', article='a', test_unit='f', multiple_suborders=0)`

Reply:
done

Reply:
how can I open this record after created it it redirect me to the order page without open the url of this record

Reply:
it returns the order_no

Reply:
@0xIslamTaha  yes but didn't open the last record